src/main/java/main/movie/model/dto/MovieSessionResponseDto.java

```diff
@@ -0,0 +1,77 @@
+package main.movie.model.dto;
+
+public class MovieSessionResponseDto {
+    private String showTime;
+    private int cinemaHallCapacity;
+    private Long cinemaHallId;
+    private String movieTitle;
+
+    public MovieSessionResponseDto() {
+    }
+
+    public String getShowTime() {
+        return showTime;
+    }
+
+    public void setShowTime(String showTime) {
+        this.showTime = showTime;
+    }
+
+    public int getCinemaHallCapacity() {
+        return cinemaHallCapacity;
+    }
+
+    public void setCinemaHallCapacity(int cinemaHallCapacity) {
+        this.cinemaHallCapacity = cinemaHallCapacity;
+    }
+
+    public Long getCinemaHallId() {
+        return cinemaHallId;
+    }
+
+    public void setCinemaHallId(Long cinemaHallId) {
+        this.cinemaHallId = cinemaHallId;
+    }
+
+    public String getMovieTitle() {
+        return movieTitle;
+    }
+
+    public void setMovieTitle(String movieTitle) {
+        this.movieTitle = movieTitle;
+    }
+
+    public static Builder newBuilder() {
+        return new MovieSessionResponseDto().new Builder();
+    }
+
+    public class Builder {
```

Comment:
why have you decided to use builder here?

if you are convinced that it is needed here, let's make this class static